rdf2hpcc/src/main/java/org/hpccsystems/rdf/rdf2hpcc/RDFHPCCWsClient.java

```diff
@@ -440,31 +441,22 @@ public String getRDFStats()
 
         try
         {
-        	if(eclstatsfile != null)
+            if(eclstatsfile != null)
         	{
         		eclstats = new String(Files.readAllBytes(Paths.get(eclstatsfile)), Charset.defaultCharset());
         	} else
         	{
         		eclstats = STATSECL;
         	}
         	
-            HPCCECLDirectClient declient = getEclDirectClient();
-            WorkunitInfo wu = new WorkunitInfo();
+            HPCCWsWorkUnitsClient wuClient = getWsWorkunitsClient();
+            WorkunitWrapper wu = new WorkunitWrapper();
             wu.setECL(eclstats + "\n output(RdfTypeStats('~" + targetHPCCFilePath + "'));");
             wu.setCluster(targetECLCluster);
-            wu.setResultLimit(HPCCECLDirectClient.noresultlimit);
+            wu.setResultLimit(100);
```

Comment:
Any reason for 100 being the result limit here?

Reply:
I have to think about that one, we HPCCEclDirect is no longer available, which takes away the ability to set "noresultlimit", if allowed, I will not set the result limit.